docs/Cluster Guides/Using_Software_on_the_Clusters.md

```diff
@@ -1,11 +1,149 @@
 # Using Software on the Clusters
 
-There are a number of different software packages available on R2 and Borah. The links below are to guides to the use of some packages on the clusters:
+There are a number of different software packages available on R2 and Borah. Below are guides to the use of some packages on the clusters:
 
-- [Singularity](https://www.boisestate.edu/rcs/singularity/)
-- [MOOSE](https://www.boisestate.edu/rcs/moose/)
-- [MATLAB](https://www.boisestate.edu/rcs/matlab/)
-- [Gaussian](https://www.boisestate.edu/rcs/cluster-guides/using-software-on-the-clusters/gaussian/)
+## Singularity
+### Singularity
+Software is sometimes containerized for ease of installation and use on the clusters, typically when there are numerous dependencies needed for the software. A container is a self-sufficient package containing an entire Linux operating system with the required software and dependencies installed inside of it. On the campus clusters, the software used to make these containers is called Singularity. If a singularity container is being built for your workflow, instructions on its use will be provided. 
+
+## MOOSE
+### MOOSE
+Getting started with MOOSE on Borah
+
+**1.** Login to Borah and type **module load petsc**. This statement must also be in your sbatch file when you run Moose. 
+
+**2.** Navigate to the directory you want to install Moose in. Your scratch directory is a common place to put it as Moose and data files can grow to be very large. Then, run the following commands on the command line:
+
+**3.** **mkdir projects**
+
+**4.** **cd projects**
+
+**5.** **git clone https://github.com/idaholab/moose.git**
+
+**6.** **cd moose**
+
+**7.** **git checkout master**
+
+**8.** **./scripts/update_and_rebuild_libmesh.sh**
+
+**9.** Step 8 may take 1-2 hours to complete. When it is finished, do **cd test** and then: 
+
+**10.** **make**
+
+**11.** **./run_tests**
+
+**12.** Remember you will need to use a sbatch file to run your jobs on Borah. An example file that you can copy and modify can be found at **/cm/shared/examples/slurm_moose.bash**
+
```

Comment:
These instructions are a little outdated. We've been recommending that folks install moose via conda: https://mooseframework.inl.gov/getting_started/installation/moose_conda_binary.html
```suggestion
```